<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWindow</class>
 <widget class="QDialog" name="DashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;welcome&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalBooksLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>totalbooks</string>
   </property>
  </widget>
  <widget class="QLabel" name="borrowedBooksLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>borrowedbooks</string>
   </property>
  </widget>
  <widget class="QLabel" name="overdueBooksLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>overduebooks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewBooksButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Records</string>
   </property>
  </widget>
  <widget class="QPushButton" name="borrowReturnButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>151</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow/Return Books</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createBookButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateReportsButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Reports</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
